test/models/item_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ItemTest < ActiveSupport::TestCase
```

Comment:
Please delete these `_test` files if you aren't going to use them.

Also, if you do want tests,  you should use Rspec like we did for other projects.  Everything under the folder `test` is for a different testing framework `ActiveSupport::Test` which is included in Rails by default but not as commonly used